back/engines/free/email_campaigns/spec/mailers/previews/email_campaigns/native_survey_not_submitted_mailer_preview.rb

```diff
@@ -7,6 +7,31 @@ class NativeSurveyNotSubmittedMailerPreview < ActionMailer::Preview
     def campaign_mail
       campaign = EmailCampaigns::Campaigns::NativeSurveyNotSubmitted.first
       idea = Idea.where.not(creation_phase: nil).order(created_at: :asc).first
+      if idea.nil?
+        project = Project.create!(
+          title_multiloc: { 'en' => 'Created for survey not submitted mail preview' },
+          description_multiloc: { 'en' => 'Created for survey not submitted mail preview' },
+          slug: 'for-survey-not-submitted-mail-preview',
+          visible_to: 'public'
+        )
+        phase = Phase.create!(
+          project: project,
+          title_multiloc: { 'en' => 'Created for survey not submitted mail preview' },
+          description_multiloc: { 'en' => 'Created for survey not submitted mail preview' },
+          start_at: Time.now - 1.day,
+          end_at: Time.now + 1.day,
+          participation_method: 'native_survey',
+          submission_enabled: true,
+          native_survey_title_multiloc: { 'en' => 'Survey' },
+          native_survey_button_multiloc: { 'en' => 'Take the survey' },
+          campaigns_settings: { 'project_phase_started' => true }
+        )
+        idea = Idea.create!(
+          project: project,
+          creation_phase: phase,
+          author: User.first
+        )
+      end
```

Comment:
The alternative is to seed this data, but I am guessing this was originally the case and someone decided to clean up the seeded data.

The problem is broken previews aren't caught by specs, but only when a preview is loaded via `http://localhost:4000/rails/mailers`

I have tried to make it easier to understand why the preview breaks when/if new validations are added, by the use of `create!` which will explicitly detail the failed validation(s) in the preview page itself.